src/Flex.php

```diff
@@ -156,7 +157,11 @@ public function activate(Composer $composer, IOInterface $io)
             }
 
             if ('create-project' === $command) {
-                $input->setInteractive(false);
+                if (class_exists(Comparer::class)) {
```

Comment:
The condition on the presence of the class is to determine if the option is present ?
If so, is not it be better to use the Composer constant like this `$this->composer::VERSION` ?

Reply:
Because the constant is not populated for snapshot versions of Composer...

Reply:
(comment added)